### DISCUSSION TITLE
Modificar valores de objeto

### DISCUSSION THREAD

**Discussion by Burstniart** [LaunchX-InnovaccionVirtual/FrontEnd-Mision#511] [Q&A] [answered]
Estoy corriendo un ciclo para que se impriman los valores al realizar el pedido con el siguiente codigo:

\<tr v-for="(fullOrder,index) in $store.getters.getAllOrders" :key="index">
            <td>{{fullOrder.flavors}}</td>
\<tr>

El detalle es que los valores se imprimen con corchetes y comillas al ser consultados del objeto de esta manera
[ "Chocolate", "Tres Leches"]
Quiero correr una funcion como: 

text.replace(/[|]|"/g, "")  

para recortar los caracteres no deseados sin modificar el objeto directamente pero no estoy seguro de donde y como colocar esta funcion. O si hay alguna otra manera.

**Comment by Burstniart** [comment, marked as answer]
Nvm nada mas le agregue toString() y funciono

**Reply by AngelCruzO** [reply]
Gracias por compartir, me sirvio bastante
